update/sync/syncAccountsDBs.go

```diff
@@ -200,21 +194,25 @@ func (st *syncAccountsDBs) tryRecreateTrie(shardId uint32, id string, trieID sta
 		}
 	}
 
-	st.setTries(shardId, id, rootHash, tries)
-
-	return true
-}
+	for hash, tr := range tries {
```

Comment:
maybe we should have a TODO to investigate if this can raise OOMs

Reply:
Added TODO.